app/views/fear_quiz/new.html.erb

```diff
@@ -1,15 +1,50 @@
-<div class="inner-page-body">
+<header class='quiz'>
   <div class="container d-flex h-100 align-items-center">
-    <div class="mx-auto text-center">
-      <h1>Fear Quiz Page</h1>
-      <p>What're you afraid of?</p>
-    <div class="quiz">
-      This is where a quiz would go.<br><br>
-      Are you afraid of:<br>
-      <li>Spiders</li>
-      <li>spiders</li>
-      <li>sPiDeRs</li>
-      <li>spdr</li>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+    <section>
+      <h1 class="mx-auto text-uppercase">Tell Me Your Fears</h1>
+
+      <input id='one' type='checkbox' />
+      <label for='one'>
+        <span></span>
+          Spiders
+        <ins><i>Spiders</i></ins>
+      </label>
+
+      <input id='two' type='checkbox' />
+      <label for='two'>
+        <span></span>
+          Heights
+        <ins><i>Heights</i></ins>
+      </label>
+
+      <input id='three' type='checkbox' />
+      <label for='three'>
+        <span></span>
+          Zombies
+        <ins><i>Zombies</i></ins>
+      </label>
+
+      <input id='four' type='checkbox' />
+      <label for='four'>
+        <span></span>
+          Death
+        <ins><i>Death</i></ins>
+      </label>
+
+      <input id='five' type='checkbox' />
+      <label for='five'>
+        <span></span>
+          Outdoors
+        <ins><i>Outdoors</i></ins>
+      </label>
+
+      <input id='six' type='checkbox' />
+      <label for='six'>
+        <span></span>
+          Lonliness
+        <ins><i>Lonliness</i></ins>
+      </label>
+
+      <br><a class="btn btn-primary js-scroll-trigger" href="/results">See My Results</a>
```

Comment:
I talked with my mentor and our quiz is technically a form. I wasn't sure if this button should be a button or a submit tag button like what we used with forms. In my mind, the results should only save if the user is logged in. Visitors can take the quiz but our system doesn't need to save their results